Cargo.toml

```
@@ -47,6 +47,7 @@ postgres-openssl = "0.5.1"
 regex = "1.10.4"
 reqwest = "0.11.22"
 rust_decimal = { version = "1.37.1", features = ["db-postgres"] }
+rust_decimal_macros = { version = "1.37.1" }
```

```suggestion
rust_decimal_macros = "1.37.1"
```
(non-blocking)
could be shortened to just this if no `features` are used